test/ui/cypress/code/environment.js

```diff
@@ -0,0 +1,28 @@
+import SemverResolver from "./semverResolver";
+
+class Environment {
+  ocpVersion() {
+    if (!Environment.__ocpVersion) {
+      Environment.__ocpVersion = new SemverResolver(Cypress.env('OCP_VERSION'))
+    }
+    return Environment.__ocpVersion
```

Comment:
It's just a single class initialization plus reading the env variable so I'm wondering if we could simplify this whole function to just `return new SemverResolver(Cypress.env('OCP_VERSION'))`. Saving the value for later might be unnecessary here. After all, in each class you initialize the Environment class once and then mostly call ocpVersion() only 1x.

Reply:
Yes. We could.

Initially, I thought of momization for this as this was a promise, but moving all that code to simple classes made the code promise'less. So, this is left over from those experiments.